kentik_api_library/pyproject.toml

```diff
@@ -10,19 +9,105 @@ check = true
 fast = true
 
 [tool.isort]
+line_length = 120
 profile = "black"
 known_local_folder = ["kentik_api", "examples", "tests"]
-line_length = 120
 
 [tool.mypy]
 ignore_missing_imports = true
 exclude = "(generated|build)/"
 
+[[tool.mypy.overrides]]
+module = "/generated/*"
+ignore_errors = true
+
 [tool.pytest.ini_options]
 testpaths = ["tests/integration", "tests/unit"]
 
-[tool.pylint]
-
-[[tool.mypy.overrides]]
-module = "/generated/*"
-ignore_errors = true
\ No newline at end of file
+[tool.pylint.message_control]
+max-line-length = 120
+ignore = "examples"
```

Comment:
While we're at it: I think that examples should have highest quality and be linted, because they might be copy-pasted by users to their code-bases.

Reply:
OK